DependencyInjection/Configuration.php

```diff
@@ -65,41 +64,19 @@ public function getConfigTreeBuilder()
                     ->end()
                 ->end()
                 ->arrayNode('document_tree')
-                    ->useAttributeAsKey('class')
-                    ->prototype('array')
-                        ->fixXmlConfig('valid_child', 'valid_children')
-                        ->children()
-                            ->arrayNode('valid_children')
-                                ->prototype('scalar')->end()
-                                ->info('class names of valid children, manage tree operations for them and hide other children')
-                            ->end()
-                            ->scalarNode('image')
-                                ->defaultValue('')
-                            ->end()
-                        ->end()
-                    ->end()
-                 ->end()
-
-                 ->scalarNode('document_tree_repository')->defaultValue('default')->end()
-
-                ->arrayNode('document_tree_defaults')
-                    ->prototype('scalar')->end()
-                ->end()
-
-                ->arrayNode('document_tree_options')
-                    ->addDefaultsIfNotSet()
+                    ->canBeEnabled()
                     ->children()
-                        ->integerNode('depth')
-                            ->defaultValue(1)
-                            ->info('Depth to which to fetch tree children when rendering the initial tree')
+                        ->scalarNode('repository_name')
+                            ->defaultValue('default')
```

Comment:
should this not be null by default? when we ask the manager-registry for the default repository, we should ask for `null` rather than the one called "default". "default" is the default name for the default repository, but afaik `null` is the correct parameter to ask for the default repo (as the name could be changed to something else)